src/index.js

```diff
@@ -81,6 +81,9 @@ export default {
   get KeyboardAwareListView() {
     return require('./components/KeyboardAwareScrollView').KeyboardAwareListView;
   },
+  get KeyboardAwareFlatList() {
```

Comment:
Now I notice we export KeyboardAwareFlatList, KeyboardAwareScroll and KeyboardAwareListView as standalone components and not as part of the Keyboard group. 
Why is that?

Reply:
We did not start migrating all of the OS libs, only `react-native-keyboard-input` and `react-native-keyboard-tracking-view`

Reply:
But still? 
Why not include them in the keyboard namespace? 
Which libraries we have left?

Reply:
I fear that someone will start using it, and I'm not sure there won't be conflicts (in native code).
We're using the `Keyboard` as is in the private lib and we're not exporting the `KeyboardAware` part (which is installed from the original by the engine).

Reply:
Oh, these are the ones I remember:
https://github.com/wix/react-native-keyboard-aware-scrollview
https://github.com/wix/react-native-autogrow-textinput

I think I left my notebook with the notes in the office :D

Reply:
>I fear that someone will start using it, and I'm not sure there won't be conflicts (in native code).
We're using the Keyboard as is in the private lib and we're not exporting the KeyboardAware part (which is installed from the original by the engine).

Im a little confused about this one.. 
let's have a quick talk about when you have time..